components/validator/l2_output_submitter.go

```diff
@@ -298,12 +338,11 @@ func submitL2OutputTxData(abi *abi.ABI, output *eth.OutputResponse, bondAmount u
 
 // submitL2OutputTx sends the l2 output submit tx to txCandidates channel to process validator's tx candidates in order.
 func (l *L2OutputSubmitter) submitL2OutputTx(data []byte) {
+	// TODO(sara): add correct accessList using `eth_createAccessList` RPC method.
+	// When you provide incorrect accessList, the transaction might be failed.
 	accessList := types.AccessList{
 		types.AccessTuple{
 			Address: l.cfg.ValidatorPoolAddr,
-			StorageKeys: []common.Hash{
-				common.HexToHash("0000000000000000000000000000000000000000000000000000000000000036"),
-			},
 		},
 	}
```

Comment:
I removed the storage key of access list. Because due to the existing access list, the transaction is confirmed but the status is confirmed as failed. This is probably because the storage key is incorrect, or all the required keys are not provided. We might to look up geth code later, but I don't think this is a critical issue now, so I've marked it TODO.

Reply:
This is strange. Could you share me with how to test this?

Reply:
When providing `accessList` that is not actually accessed, the transaction may not be executed due to exceeding the estimated gas limit. I fixed incorrect storage key of output submit tx, and added transaction receipt fail status handling in `txmgr`.